style.css

```diff
@@ -78,14 +99,43 @@ p {
 
 ul li {
   font-weight: 600;
-  font-family: 'Inter', sans-serif;
   list-style: none;
 }
 
 .ruby {
   justify-self: center;
 }
 
+.multi-story-5 h2 {
+  font-weight: 700;
+  font-family: "Roboto", sans-serif;
+  color: rgba(147, 144, 255, 1);
+  line-height: 44px;
+}
+
+/* input[name] */
+
+.name,
+.email {
+  padding-bottom: 1em;
+}
+
+input[type="text"],
+input[type="email"] {
+  font-weight: 400;
+  font-family: "Inter", sans-serif;
+  width: 19.5em;
+  height: 3em;
```

Comment:
- [ ] OPTIONAL: This fixed-width here is messing up the proper alignment of form elements. Please make a change here whenever you have spare time and fix this issue. It is not a requirement for the project submission though, as you have passed most of the mandatory requirements and this can be optionally improved. See the attached screenshot image for reference. 
![image](https://user-images.githubusercontent.com/64894927/121427466-6bb43800-c97d-11eb-8b3b-b0e8405c1871.png)    
![image](https://user-images.githubusercontent.com/64894927/121427531-7d95db00-c97d-11eb-8323-eed3e91065a8.png)